ui/models.py

```diff
@@ -76,7 +77,7 @@ def all_admin(self, user):
             models.Q(owner=user)).distinct()
 
 
-class Collection(models.Model):
+class Collection(TimestampedModel):
```

Comment:
The default ordering for `Collection` is `-created_at`, which is probably still fine, but maybe check with @pdpinch to see if he'd prefer `updated_at` instead.

Reply:
I think it would affect the order of the collections returned, so instead of the most recently created collections at the top, it would be the most recently updated.